_posts/2019-06-25-lambda-quickstart.md

```diff
@@ -0,0 +1,320 @@
+---
+title: How to configure OpenFaaS functions to execute on AWS Lambda
+description: In a follow up article Edward Wilde shows how to configure OpenFaaS functions to execute on AWS Lamda using the new faas-lambda provider.
+date: 2019-06-25
+image: /images/faas-quickstart/backdrop.jpg
+categories:
+  - lambda
+  - automation
+  - tutorial
+author_staff_member: ed
+dark_background: true
+---
+
+In this post we are going to follow the steps necessary to configure OpenFaaS to execute function on AWS Lambda using 
+the new faas-lambda provider. See the previous blog post [announcing support for AWS Lambda] for an overview of the 
+technology.
+
+
+### Pre-requisites
+To follow the steps in this article it is only necessary to have a Kubernetes cluster with Helm installed.
+
+- [Kubernetes cluster](https://kubernetes.io/docs/setup/)
+- [Helm installed](https://github.com/openfaas/faas-netes/blob/master/HELM.md)
+
+### 1. Get an early access token
+`faas-lambda` is currently in early access, please you the link below to get your access token
+
+- [Create an access token](https://ewilde.o6s.io/faas-lambda-jwt-page)
```

Comment:
I don't think the function is finished yet, I gave feedback several weeks ago on 1:1 DM with you on Slack.

The function needs to post back to itself and use two different routes:
1) serving the redirect link
2) showing the result & storing a cookie
3) if a cookie isn't stored then people can't refresh or navigate back without getting an error

This was going to move to a GitHub org with a better name. OpenFaaS incubator perhaps?

Reply:
Any thoughts? I'm happy to help if needed.

Reply:
Happy to move the functions to the incubator.

Not sure on the feedback, I think it's good enough for now.